sdk/keyvault/azure-keyvault-secrets/azure/keyvault/secrets/_shared/challenge_auth_policy.py

```diff
@@ -74,6 +72,12 @@ def _get_challenge_request(request):
 class ChallengeAuthPolicy(ChallengeAuthPolicyBase, HTTPPolicy):
     """policy for handling HTTP authentication challenges"""
 
+    # pylint:disable=useless-super-delegation
```

Comment:
is this pylint pragma necessary now that the initializer does work? There's some other places where the same comment applies

Reply:
It isn't, thanks; I'll remove it.